test/integration/local/processing/process-payment-requests.test.js

```diff
@@ -340,4 +341,58 @@ describe('process payment requests', () => {
     expect(holds.length).toBe(0)
     expect(mockSendMessage).not.toBeCalled()
   })
+
+  test('should process reduction request and create completed request if handleSchemeClosures equals true', async () => {
```

Comment:
Currently this test checks if an AP element appears for a recovery.   The answer to that will always be yes whether this change was added or not.

Suggest that this test verifies that no AR completed payment request is created for a recovery if the FRN is closed and handling scheme closures is enabled.

Then add similar tests to make sure an AR completed request is created if handleSchemeClosures is set to false and one for if it's true but the date hasn't yet happened.

Reply:
tests refined and added